.gitignore

```diff
@@ -3,3 +3,5 @@ vendor/
 share/
 build/
 keys/
+.idea/
```

Comment:
Your ide is not a problem of this lib. Add this globally in you home gitignore

Reply:
okay, I'll fix that